src/test/java/picard/sam/PipedDataTest.java

```diff
@@ -0,0 +1,67 @@
+package picard.sam;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.cmdline.CommandLineProgramTest;
+
+public class PipedDataTest extends CommandLineProgramTest {
+
+    private String classPath = "\"" + System.getProperty("java.class.path") + "\" ";
+
+
+    @Override
+    public String getCommandLineProgramName() {
+        return SortSam.class.getSimpleName();
```

Comment:
I guess that this is wrong...maybe an exception saying that "this shouldn't be called" is better than assigning a name for a random tool...

Reply:
Do we need to extend `CommandLineProgramTest` at all? It doesn't seem a good fit if it forces picking one tool name (but I'm not sure of all the magic it provides).

Reply:
I think that in this case the "magic" (making command line with arguments and the tool name, or running the tool using `Main`) is not used anywhere, so probably is better to don't extend...

Reply:
This is a remnant of an older revision that I forgot to delete.